src/NuGet/Microsoft.Net.Compilers.Toolset/Microsoft.Net.Compilers.Toolset.Package.csproj

```
@@ -15,7 +15,7 @@
 
       The supported mechanism for providing new compilers in a build enviroment is updating to the newer .NET SDK or Visual Studio Build Tools SKU.
 
-      This package requires either MSBuild 16.3 and .NET Desktop 4.7.2+ or .NET Core 2.1+
+      This package requires either MSBuild 16.3 and .NET Framework 4.7.2+ or .NET Core 3.1+.
```

I'm not sure about the MSBuild version requirement.

```suggestion
      This package requires either MSBuild 16.3 and .NET Framework 4.7.2+ or .NET 6.0+.
```